jujugui/static/gui/src/test/test_app.js

```diff
@@ -53,7 +53,7 @@ describe('App', function() {
   var jujuConfig, container, testUtils, yui;
 
   before(done => {
-    YUI(GlobalConfig).use(['juju-gui', 'juju-tests-utils',], function(Y) {
```

Comment:
If `'juju-gui'` is included here as well as in the child `describe`s it would sometimes cause conflicts and time out.